PackageExplorer/app.config

```diff
@@ -2,10 +2,70 @@
 <configuration>
     <configSections>
         <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+            <section name="PackageExplorer.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
             <section name="NuGetPackageExplorer.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
```

Comment:
I think the `NuGetPackageExplorer` settings should be removed. See https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/pull/404#discussion_r183957221

---

The settings were originally in the `PackageExplorer` namespace (https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/commit/b698085a15608b978c6b63db4fbf55c68fba187d#diff-0af6157b86a5943f59ec5be95d647bb7) but this got changed in https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/commit/b1729afb016588acf6b6f75959f6b65bcd60b519#diff-0af6157b86a5943f59ec5be95d647bb7 and with https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/commit/d9d37f8ff097299b11c0a2e0ffb966d7d14d8d72#diff-0af6157b86a5943f59ec5be95d647bb7 it got changed back to `PackageExplorer` but without updating the `app.config` file.